### PR TITLE
ui modifications found in ui review

### PR DESCRIPTION
**Pull request by dagheyman**

### CODE REVIEW

File: ykman-gui/qml/PivGenerateCertificateWizard.qml [hunk 1]
```diff
@@ -96,13 +96,12 @@ ColumnLayout {
...
-                        "Overwrite?",
-                        firstMessageTemplate.arg(slot.name).arg(slot.hex)
-                        + " This action cannot be undone! Are you sure you want to continue?",
-                        function () {
-                            finish(true)
-                        })
+            confirmationPopup.show("Overwrite?", firstMessageTemplate.arg(
+                                       slot.name).arg(slot.hex) + "
+
+Do you want to continue?", function () {
```
[the remark this answers is not in the document]
Comment (dagheyman) [reply ykman-gui/qml/PivGenerateCertificateWizard.qml]: Yes, it felt redundant.

File: ykman-gui/qml/ChangePinView.qml [hunk 2]
```diff
@@ -121,16 +118,13 @@ ColumnLayout {
...
             }
-            TextField {
+            CustomTextField {
                 id: newPin
                 Layout.fillWidth: true
-                background.width: width
                 echoMode: TextInput.Password
                 ToolTip.delay: 1000
                 ToolTip.visible: hovered
                 ToolTip.text: newPinTooltip
```
[the remark this answers is not in the document]
Comment (dagheyman) [reply ykman-gui/qml/ChangePinView.qml]: Agreed, I'll look into that.

File: ykman-gui/qml/PivGenerateCertificateWizard.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (emlun) [reply ykman-gui/qml/PivGenerateCertificateWizard.qml]: :+1:

File: ykman-gui/qml/ChangePinView.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (emlun) [reply ykman-gui/qml/ChangePinView.qml]: I can do it :)

[the remark this answers is not in the document]
Comment (emlun) [reply ykman-gui/qml/ChangePinView.qml]: #151

[the remark this answers is not in the document]
Comment (dagheyman) [reply ykman-gui/qml/ChangePinView.qml]: awesome, merged

**Pull request merged**